### PR TITLE
Update hugepages doc

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Starting from the OpenShift 4.6 the PAO can configure hugepages with the
different sizes on the node. This functionality should be reflected under
the documentation.

### CODE REVIEW

File: pkg/apis/performance/v1/performanceprofile_types.go [hunk 1]
```diff
@@ -16,6 +16,8 @@ type PerformanceProfileSpec struct {
 	// CPU defines a set of CPU related parameters.
 	CPU *CPU `json:"cpu,omitempty"`
 	// HugePages defines a set of huge pages related parameters.
+	// It possible to define huge pages with the different size, for example you can in the same time
```
Comment (fromanirh) [inline pkg/apis/performance/v1/performanceprofile_types.go:19]: It *is* possible?

File: pkg/apis/performance/v1/performanceprofile_types.go [hunk 2]
```diff
@@ -16,6 +16,8 @@ type PerformanceProfileSpec struct {
...
+	// define 1G and 2M and the performance-addon-operator will configure both sizes on the node.
```
Comment (fromanirh) [inline pkg/apis/performance/v1/performanceprofile_types.go:20]: "you can define at the same time"?

File: pkg/apis/performance/v1/performanceprofile_types.go (hunk 1 above)
Comment (yanirq) [reply pkg/apis/performance/v1/performanceprofile_types.go:19]: Suggestion: 
It is possible to set huge pages with multiple size values at the same time. For example, hugepages can be set with 1G and 2M, both values will be set on the node by the performance-addon-operator.

Comment (cynepco3hahue) [reply pkg/apis/performance/v1/performanceprofile_types.go:19]: done

File: pkg/apis/performance/v1/performanceprofile_types.go (hunk 2 above)
Comment (cynepco3hahue) [reply pkg/apis/performance/v1/performanceprofile_types.go:20]: done

**Comment by cynepco3hahue** [comment]
@fromanirh @yanirq I update the comment, I also added some important note regarding the default hugepages size.

**Pull request merged**